src/neo4j/cypher-queries/company.js

```diff
@@ -153,7 +163,7 @@ const getShowQuery = () => `
 			(entity:Person OR entity:Company) AND
 			entityRel.creditedCompanyUuid IS NULL
 
-	UNWIND (CASE WHEN entityRel IS NOT NULL AND EXISTS(entityRel.creditedMemberUuids)
+	UNWIND (CASE WHEN entityRel IS NOT NULL AND entityRel.creditedMemberUuids IS NOT NULL
```

Comment:
This message would appear for this section's former state:

`This feature is deprecated and will be removed in future versions.`
``The property existence syntax `… exists(variable.property)` is deprecated, please use `variable.property IS NOT NULL` instead.``